app/javascript/angular/code/services/google/_map.js

```diff
@@ -95,6 +95,20 @@ export const map = (
     },
 
     fitBounds: function(bounds, zoom) {
+      const fnc = (latLngBounds, zoom) => () =>
```

Comment:
not a fan of these `fnc` functions, even though they are used near their definitions

Reply:
If you have a better idea please share!

Reply:
would if I had, sorry